packages/frontend/src/components/InboxItem/InboxItem.tsx

```diff
@@ -95,15 +96,26 @@ export const InboxItem = ({
   isUnread = false,
   linkTo,
 }: InboxItemProps): JSX.Element => {
+  const { selectedItemCount } = useSelectedDialogs();
```

Comment:
Det virker ikke for meg som at du bruker `selectedItemCount` annet enn til å som verdien er større enn 0, både her og i PageLayout. Kanskje du skal i stedet bare eksponere `inSelectionMode` fra hook, og omskrive til å bruke den?